rustls/tests/api.rs

```diff
@@ -3319,7 +3319,7 @@ mod test_quic {
 
     #[test]
     fn test_quic_handshake() {
-        fn equal_packet_keys(x: &quic::PacketKey, y: &quic::PacketKey) -> bool {
+        fn equal_packet_keys(x: &Box<dyn quic::PacketKey>, y: &Box<dyn quic::PacketKey>) -> bool {
```

Comment:
Should probably call these with `&dyn` directly rather than requiring the `Box` wrapping?